packages/wallets/src/components/AppCardBadge/AppCardBadge.scss

```diff
@@ -0,0 +1,24 @@
+.wallets-app-card {
+    &__badge {
+        display: flex;
+        width: fit-content;
+        padding: 0.4rem 0.8rem;
+        justify-content: center;
+        align-items: center;
+        height: 2.2rem;
+        border-radius: 0 0.8rem;
+
+        @include mobile {
+            height: 1.6rem;
+            padding-block: 0.2rem;
+        }
+
+        &--demo {
```

Comment:
minor - typically modifiers are best to put at top level of component, 
e.g. 

.wallets-app-card--demo
.wallets-app-card--real

unless there is reason for that, e.g. displaying two badges of different types (so then it needs to be on element level)

Reply:
actually, after having closer look, the name of the class should be:

.wallets-app-card-badge {
   &--demo { ... }
   &--real { ... } 
}